FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -0,0 +1,1457 @@
+                              Command_t * pxCommand );
+
+/**
+ * @brief Add a command to the global command queue.
+ *
+ * @param[in] pxCommand Pointer to command to copy to queue.
+ */
+static void prvAddCommandToQueue( Command_t * pxCommand );
+
+/**
+ * @brief Copy an incoming publish to a response queue.
+ *
+ * @param[in] pPublishInfo Info of incoming publish.
+ * @param[in] pResponseQueue Queue to which the publish is copied.
+ */
+static void prvCopyPublishToQueue( MQTTPublishInfo_t * pPublishInfo,
+                                   QueueHandle_t pResponseQueue );
+
+/**
+ * @brief Process a #Command_t.
+ *
+ * @param[in] pxCommand Pointer to command to process.
+ *
+ * @return return status of MQTT library API call.
+ */
+static MQTTStatus_t prvProcessCommand( Command_t * pxCommand );
+
+/**
+ * @brief Dispatch incoming publishes and acks to response queues and
+ * command callbacks.
+ *
+ * @param[in] pMqttContext MQTT Context
+ * @param[in] pPacketInfo Pointer to incoming packet.
+ * @param[in] pDeserializedInfo Pointer to deserialized information from
+ * the incoming packet.
+ */
+static void prvSubscriptionManager( MQTTContext_t * pMqttContext,
+                                    MQTTPacketInfo_t * pPacketInfo,
+                                    uint16_t packetIdentifier,
+                                    MQTTPublishInfo_t * pPublishInfo );
+
+/**
+ * @brief Process commands from the command queue in a loop.
+ *
+ * This demo requires a process loop command to be enqueued before calling this
+ * function, and will re-add a process loop command every time one is processed.
+ * This demo will exit the loop after receiving an unsubscribe operation.
+ */
+static void prvCommandLoop();
+
+/**
+ * @brief Common callback for commands in this demo.
+ *
+ * This callback marks the command as complete and notifies the calling task.
+ *
+ * @param[in] pxContext Context of the initial command.
+ */
+static void prvCommandCallback( CommandContext_t * pxContext );
+
+/**
+ * @brief The task used to create various publish operations.
+ *
+ * @param[in] pvParameters Parameters as passed at the time of task creation. Not
+ * used in this example.
+ */
+void prvPublishTask( void * pvParameters );
+
+/**
+ * @brief The task used to wait for incoming publishes.
+ *
+ * @param[in] pvParameters Parameters as passed at the time of task creation. Not
+ * used in this example.
+ */
+void prvSubscribeTask( void * pvParameters );
+
+/**
+ * @brief The main task used in the MQTT demo.
+ *
+ * @param[in] pvParameters Parameters as passed at the time of task creation. Not
+ * used in this example.
+ */
+static void prvMQTTDemoTask( void * pvParameters );
+
+/**
+ * @brief The timer query function provided to the MQTT context.
+ *
+ * @return Time in milliseconds.
+ */
+static uint32_t prvGetTimeMs( void );
+
+/*-----------------------------------------------------------*/
+
+/* Global MQTT context. */
```

Comment:
Please follow a consistent style in comments for the static variables. You have used doxygen style comments from line 395. I think it will be good to follow that style.